packages/client-core/src/components/InviteToast/InviteSnackbarActions.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/ir-engine/ir-engine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Infinite Reality Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Infinite Reality Engine team.
+
+All portions of the code written by the Infinite Reality Engine team are Copyright © 2021-2023 
+Infinite Reality Engine. All Rights Reserved.
+*/
+
+import { InviteType } from '@ir-engine/common/src/schema.type.module'
+import { useMutableState } from '@ir-engine/hyperflux'
+import Button from '@ir-engine/ui/src/primitives/tailwind/Button'
+import { first } from 'lodash'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { InviteService, InviteState } from '../../social/services/InviteService'
+
+type Props = {
+  closeSnackbar: () => void
+}
+
+const InviteSnackbarActions = ({ closeSnackbar }: Props) => {
```

Comment:
we can add `newestInvite` as prop rather than getting it from state once `inviteAction` is removed from notificationService

Reply:
The InviteSnackbarActions is being called from the service and with the current implementation I can only show the content which is the message, I cannot send props to the notification service service

https://github.com/ir-engine/ir-engine/blob/0214f222d701c96b66f384e51831823437d0ac7d/packages/client-core/src/common/services/NotificationService.tsx#L62